src/adaptors/tcp_adaptor.c

```diff
@@ -1527,8 +1527,11 @@ static void qdr_del_tcp_connection_CT(qdr_core_t *core, qdr_action_t *action, bo
         qdr_tcp_connection_t *conn = (qdr_tcp_connection_t*) action->args.general.context_1;
         if (conn->in_list) {
             DEQ_REMOVE(tcp_adaptor->connections, conn);
-            qd_log(tcp_adaptor->log_source, QD_LOG_DEBUG, "[C%"PRIu64"] qdr_del_tcp_connection_CT %s (%zu)",
-                   conn->conn_id, conn->config.host_port, DEQ_SIZE(tcp_adaptor->connections));
+            qd_log(tcp_adaptor->log_source, QD_LOG_DEBUG,
+                   "[C%"PRIu64"] qdr_del_tcp_connection_CT %s (%zu) bytes_in=%"PRIu64", bytes_out=%"PRId64
+                   ", opened_time=%"PRId64", last_in_time=%"PRId64", last_out_time=%"PRId64"",
```

Comment:
Also the **(%zu)** value could stand clarification. Maybe **(tcp connections %zu)**